pkgs/by-name/ra/raffi/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "raffi";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "chmouel";
+    repo = "raffi";
+    rev = "v${version}";
+    hash = "sha256-i4PM82vGb9Z2pwW006114/h9crokVLUpLxNjr7tgAU8=";
+  };
+
+  cargoHash = "sha256-DS56H2XjEgdXC9TKLjwyfLpFHB9dUThhr8pNFEJuAZE=";
+
```

```suggestion

  nativeBuildInputs = [ makeBinaryWrapper ];

  postFixup = ''
    wrapProgram $out/bin/raffi \
      --prefix PATH : ${lib.makeBinPath ([ fuzzel ] ++ additionalPrograms)}
  '';

```
as this directly calls fuzzel [here](https://github.com/chmouel/raffi/blob/daa50934cfb5da22dc249fe7e0d4034f4e4b8611/src/main.rs#L150) and seems to regularly call other external executables, this should be wrapped with them accordingly

---

Thanks for reviewing - I was wondering about this but couldn't articulate it into words. I'm also going to make it so that I can pass in a custom version of `fuzzel` package or default to the existing one.